Sources/Commands/SwiftPackageTool.swift

```diff
@@ -489,7 +489,7 @@ extension SwiftPackageTool {
             case .set(let value):
                 guard let toolsVersion = ToolsVersion(string: value) else {
                     // FIXME: Probably lift this error defination to ToolsVersion.
-                    throw ToolsVersionLoader.Error.malformedToolsVersion(specifier: value, currentToolsVersion: .currentToolsVersion)
+					throw ToolsVersionLoader.Error.malformedToolsVersionSpecification(.versionSpecifier(.isMisspelt(value)))
```

Comment:
Nitpick:  the indentation here looks off.

Reply:
Ah, seems like tabs instead of spaces.

Reply:
Thanks for noticing this. I fixed it in commit c6e5e9b52b27f8ce8cbb37e89d628b8f2caf03a7.